src/merge.js

```diff
@@ -0,0 +1,3 @@
+import merge from 'lodash.merge';
+
+export default merge;
```

Comment:
I'm talking about `import merge from 'lodash.merge';`. I guess I shouldn't comment in my state :beer: :grin:.